build.gradle

```diff
@@ -6,7 +6,7 @@ apply plugin: 'eclipse'
 apply plugin: 'idea'
 
 group = 'com.github.hiroyuki-sato'
-version = '0.1.4'
+version = '0.1.5'
```

Comment:
Please keep the current version. (Revert this modification). I'll bump version later.